stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -26,7 +26,7 @@ public class _AnyTensorHandle {
   /// - Note: The compiler knows that `_AnyTensorHandle` has a single stored
   /// property, and assumes that this is it. Changing the design of
   /// `TensorHandle` will require tweaking the compiler.
-  @usableFromInline let cTensorHandle: CTensorHandle
+  public let cTensorHandle: CTensorHandle
```

Comment:
There's no good solutions to this other than prepending an underscore to it.

Reply:
I prepended an underscore.